.travis.yml

```diff
@@ -17,7 +20,15 @@ script:
   - 'if [ -n "${BUILD-}" ]; then npm run lint ; fi'
   - 'if [ "${TEST-}" = true ]; then npm run tests-only ; fi'
   - 'if [ -n "${KARMA-}" ]; then npm run tests-karma ; fi'
-  - 'if [ -n "${COVERAGE-}" ] && [ "${TRAVIS_BRANCH-}" = "master" ]; then npm run cover; cat ./coverage/lcov.info | ./node_modules/.bin/coveralls ; fi'
+  - 'if [ -n "${COVERAGE-}" ]; then npm run cover; cat ./coverage/lcov.info | ./node_modules/.bin/coveralls ; fi'
```

Comment:
The master branch restriction needs to remain applying to the coveralls binary command.

Reply:
**Agree**. Was changed it only so we could leverage the Coverage using our branch with our Travis account -- show how the results look in Testspace.